<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>731</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Gestion collaborateur</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>611</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ajouter un colloborateur</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>361</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>ajout</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>codecolloborateur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_codecolloborateur_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>478</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ll</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_company_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>company</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>afficher un colloborateur</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>431</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>supprimer un colloborateur</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>codecolloborateur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_codecolloborateur_supp">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>modifier un colloborateur</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>361</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>modifier</string>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>codecolloborateur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_codecolloborateur_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>kkk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_company_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ll</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>company</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_mod">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>rrr</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Gestion events</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>661</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Ajouter events</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>361</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>ajout</string>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>codeevenement</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>lieu</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_codecolloborateur_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ll</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>lk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_company_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>date-events</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>afficher events</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>431</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>supprimer events</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>codeevenement</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_codecolloborateur_supp_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>modifier events</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>361</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>modifier</string>
       </property>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>codeevenement</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>lieu</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_codecolloborateur_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>kkk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_company_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ll</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>date_events</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_mod_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>rrr</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
